.github/workflows/neard_custom_release.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           if [ -z "$BRANCH" ]; then
             BRANCH=$(git branch -r --contains=${{ github.ref_name }} | head -n1 | cut -c3- | cut -d / -f 2)
           fi
-          aws s3 cp --acl public-read latest s3://build.nearprotocol.com/nearcore/$(uname)/${BRANCH}/latest
+          aws s3 cp --acl public-read latest s3://build.nearprotocol.com/nearcore/$(uname)/${{ github.event.inputs.release }}/${BRANCH}/latest
```

Comment:
can we reverse the order of branch and release type?
.../statelessnet_forknet/stateleness-release/...